CONTRIBUTING.rst

```diff
@@ -102,17 +102,25 @@ It's easy to release a version of CumulusCI to GitHub and PyPI! First, create a
 
 Make the necessary changes to prepare the new release:
 
-    1. Update the version in ``cumulusci/version.txt``
-    2. Update the release notes in ``HISTORY.rst``. Information in our project history is derived from Pull Request notes found in GitHub and dating back to our previous release.
+#. Update the version in ``cumulusci/version.txt``
+#. Update the release notes in ``HISTORY.rst`` 
+    #. Navigate to the latest commits on the ``main`` branch `here <https://github.com/SFDO-Tooling/CumulusCI/commits/main>`_.
+    #. Open all merge commits dating back to the previous release.
+    #. Aggregate all content under the "Changes" heading of each of the pull requests under the new entry in the ``HISTORY.rst`` file. 
```

Comment:
There are multiple headings for a reason. Entries under "Critical changes" or "Issues closed" should go under those headings in the aggregated release notes.

Reply:
Good call. I've gone ahead and mentioned all headings that we should aggregate from.